flink-optimizer/src/main/java/org/apache/flink/optimizer/operators/GroupReduceWithCombineProperties.java

```diff
@@ -121,6 +121,9 @@ public SingleInputPlanNode instantiate(Channel in, SingleInputNode node) {
 			Channel toReducer = new Channel(combiner);
 			toReducer.setShipStrategy(in.getShipStrategy(), in.getShipStrategyKeys(),
 									in.getShipStrategySortOrder(), in.getDataExchangeMode());
+			if (in.getShipStrategy() == ShipStrategyType.PARTITION_RANGE) {
```

Comment:
In this case, a sourceNode-->targetNode would be transferred into sourceNode --> combineNode --> targetNode, if it's range partition between sourceNode and targetNode, the data distribution is forget to copy to the new created channel.